server/src/main/java/org/elasticsearch/action/admin/indices/validate/query/ValidateQueryRequest.java

```diff
@@ -179,4 +181,11 @@ public String toString() {
         return "[" + Arrays.toString(indices) + "]" + Arrays.toString(types) + ", query[" + query + "], explain:" + explain +
                 ", rewrite:" + rewrite + ", all_shards:" + allShards;
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.field("query");
+        query.toXContent(builder, params);
+        return builder;
```

Comment:
can you make it a ToXContentObject please instead?

Reply:
In this case, I should add `builder.startObject` in the `toXContent` method, right?

Reply:
yea startObject and endObject, see #31155 also